common/core/src/main/java/com/vmware/connectors/common/payloads/response/Cards.java

```diff
@@ -13,6 +13,7 @@
 /**
  * Represents a collection of {@link Card}.
  */
+@SuppressWarnings("PMD.AvoidFieldNameMatchingTypeName")
```

Comment:
I have suppressed this PMD violation. We might have to refactor this in the future. We might want to avoid naming class and variables with the same name.